build-coordinator/pom.xml

```diff
@@ -45,6 +45,10 @@
     </dependency>
     <dependency>
       <groupId>org.jboss.pnc</groupId>
+      <artifactId>rest-model</artifactId>
```

Comment:
It should be used also by bpm and it is used from our internal modules too (buildCoordinator).